search-setup/src/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const indicatorSearchIndexName = "indicator-search-index";
+export const geographySearchIndexName = "geography-search-index";
+export const geographySearchSuggesterName = "geographySuggester";
```

Comment:
If these are constants should these be capitalised? 

How about an enum instead though. Might be a nice way too?